src/hyp3_isce2/burst.py

```diff
@@ -380,3 +380,40 @@ def get_burst_params(scene_name: str) -> BurstParams:
         polarization=results[0].properties['polarization'],
         burst_number=results[0].properties['burst']['burstIndex'],
     )
+
+
+def validate_bursts(reference_name, secondary_name):
+    """Check whether the reference and secondary bursts are valid.
+
+    Args:
+        reference_scene: The reference burst name.
+        secondary_scene: The secondary burst name.
+
+    Returns:
+        None
+    """
+
+    ref_split = reference_name.split('_')
+    sec_split = secondary_name.split('_')
+
+    ref_burst_id = ref_split[1]
+    sec_burst_id = sec_split[1]
+    ref_polarization = ref_split[4]
+    sec_polarization = sec_split[4]
+
+    if ref_burst_id != sec_burst_id:
+        raise ValueError(
+            f'The reference and secondary bursts do not share a common BurstID: {ref_burst_id} and {sec_burst_id}.'
```

Comment:
I think this wording makes it more clear that the two values are the reference and secondary burst IDs.
```suggestion
            f'The reference and secondary burst IDs are not the same: {ref_burst_id} and {sec_burst_id}.'
```